whitepapers/ingest/Graph.md

```diff
@@ -0,0 +1,29 @@
+# MutableGraph [NOT IMPLEMENTED]
```

Comment:
While I hate to bring this up again but why did we decide against networkX? It [implements all the basic methods required, add_edge, remove_edge, add_node, remove_node, etc](https://networkx.readthedocs.io/en/stable/tutorial/tutorial.html#adding-attributes-to-graphs-nodes-and-edges) and it also has a [topological sort built into it](http://networkx.readthedocs.io/en/stable/reference/generated/networkx.algorithms.dag.topological_sort.html?highlight=sort).
To top it all of, [it doesn't provide it's own nodes or edges](https://networkx.readthedocs.io/en/stable/tutorial/tutorial.html#what-to-use-as-nodes-and-edges), we can us whatever objects we'd like.

@laurenbarker would you mind weighing in?